go_game/components/Game.tsx

```diff
@@ -131,15 +110,17 @@ const GoComponent: React.FunctionComponent<GameProps<IState>> = (props: GameProp
   const rows = [0, 1, 2, 3, 4, 5, 6, 7, 8];
   const cols = [0, 1, 2, 3, 4, 5, 6, 7, 8];
 
-  let hintLine = null;
+  let hintDot = null;
   if (showHint && riddleData) {
     let style: ViewStyle = {};
     if (riddleData.startsWith('r')) {
-      style = {...styles.hintLineRow};
-      const row = Number(riddleData.charAt(1));
-      style.top = 100 / 6.1 + row * (100 / 9) + '%';
+      style = {...styles.hintLineDot};
+      const {row, col}: {row: number; col: number} = riddleHints(riddleData);
+      style.top = 100 / 7.9 + (row - 1) * (100 / 9) + '%';
+      style.left = 100 / 7.9 + (col - 1) * (100 / 9) + '%';
     } else throw new Error('Illegal riddleData=' + riddleData);
-    hintLine = <View style={style} />;
+    // hintDot = <View style={style}/>
```

Comment:
Would remove unnecessary comments at the time of a PR.

Reply:
Missed it, and found one more.
Already deleted them for the next PR, thanks for pointing it out!